website/content/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways.mdx

```diff
@@ -7,12 +7,10 @@ description: |-
 
 # WAN Federation via Mesh Gateways
 
--> **1.8.0+:** This feature is available in Consul versions 1.8.0 and higher
```

Comment:
This was moved into the requirements section. Is there a reason it needs to stay here?

Reply:
None that I can see -- I think that's a fine change.